tests/ingresstests/l7_rest_test.go

```diff
@@ -35,6 +35,38 @@ import (
 	"k8s.io/apimachinery/pkg/util/intstr"
 )
 
+type IngressTestObject struct {
```

Comment:
this object is added and used to simplify the function signature of SetUpIngressForCacheSyncCheck 

combined with FillParams it allow adding default values to the function